### ISSUE TITLE
📐 Theory — 2026-04-04 (38 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#48]
### 🏆 Oral

**1. [Generalized Linear Mode Connectivity for Transformers](https://openreview.net/forum?id=KurYdcCbjv)**
  *Alexander Theus, Alessandro Cabodi, Sotiris Anagnostidis et al.*
  📍 NeurIPS 2025 oral
  > Understanding the geometry of neural network loss landscapes is a central question in deep learning, with implications for generalization and optimization. A striking phenomenon is $\textit{linear ...

**2. [Implicit Regularization for Tubal Tensor Factorizations via Gradient Descent](https://openreview.net/forum?id=2GmXJnyNM4)**
  *Santhosh Karnik, Anna Veselovska, Mark Iwen et al.*
  📍 ICML 2025 oral
  > We provide a rigorous analysis of implicit regularization in an overparametrized tensor factorization problem beyond the lazy training regime. For matrix factorization problems, this phenomenon has...

**3. [Fundamental Bias in Inverting Random Sampling Matrices with Application to Sub-sampled Newton](https://openreview.net/forum?id=LwQGRGJTHw)**
  *Chengmei Niu, Zhenyu Liao, Zenan Ling et al.*
  📍 ICML 2025
  > A substantial body of work in machine learning (ML) and randomized numerical linear algebra (RandNLA) has exploited various sorts of random sketching methodologies, including random sampling and ra...

**4. [Optimistic Task Inference for Behavior Foundation Models](https://openreview.net/forum?id=m5byThUSNE)**
  *Thomas Rupf, Marco Bagatella, Marin Vlastelica et al.*
  📍 ICLR 2026
  > Behavior Foundation Models (BFMs) are capable of retrieving high-performing policy for any reward function specified directly at test-time, commonly referred to as zero-shot reinforcement learning ...

**5. [A Representer Theorem for Hawkes Processes via Penalized Least Squares Minimization](https://openreview.net/forum?id=gJjRdLG5MY)**
  *Hideaki Kim, Tomoharu Iwata*
  📍 ICLR 2026
  > The representer theorem is a cornerstone of kernel methods, which aim to estimate latent functions in reproducing kernel Hilbert spaces (RKHSs) in a nonparametric manner. Its significance lies in c...

**6. [Overparametrization bends the landscape: BBP transitions at initialization in simple Neural Networks](https://openreview.net/forum?id=xDLE5n3x9Y)**
  *Brandon Livio Annesi, Dario Bocchi, Chiara Cammarota*
  📍 ICLR 2026
  > High-dimensional non-convex loss landscapes play a central role in the theory of Machine Learning. Gaining insight into how these landscapes interact with gradient-based optimization methods, even ...

**7. [Non-Asymptotic Analysis of (Sticky) Track-and-Stop](https://openreview.net/forum?id=vebqP5aioj)**
  *Riccardo Poiani, Martino Bernasconi, Andrea Celli*
  📍 ICLR 2026
  > In pure exploration problems, a statistician sequentially collects information to answer a question about some stochastic and unknown environment. The probability of returning a wrong answer should...

**8. [From Markov to Laplace: How Mamba In-Context Learns Markov Chains](https://openreview.net/forum?id=kmK3WSCOCT)**
  *Marco Bondaschi, Nived Rajaraman, Xiuying Wei et al.*
  📍 ICLR 2026
  > While transformer-based language models have driven the AI revolution thus far, their computational complexity has spurred growing interest in viable alternatives, such as structured state space se...

**9. [AutoEP: LLMs-Driven Automation of Hyperparameter Evolution for Metaheuristic Algorithms](https://openreview.net/forum?id=hit3hGBheP)**
  *Zhenxing Xu, Yizhe Zhang, Weidong Bao et al.*
  📍 ICLR 2026
  > Dynamically configuring algorithm hyperparameters is a fundamental challenge in computational intelligence. While learning-based methods offer automation, they suffer from prohibitive sample comple...

**10. [On The Surprising Effectiveness of a Single Global Merging in Decentralized Learning](https://openreview.net/forum?id=zrFnwRHuQo)**
  *Tongtian Zhu, Tianyu Zhang, Mingze Wang et al.*
  📍 ICLR 2026
  > Decentralized learning provides a scalable alternative to parameter-server-based training, yet its performance is often hindered by limited peer-to-peer communication. 
In this paper, we study how ...

### ⭐ Spotlight

**11. [Convergence Rates of Constrained Expected Improvement](https://openreview.net/forum?id=Dn4He1IrUT)**
  *Haowei Wang, Jingyi Wang, Zhongxiang Dai et al.*
  📍 NeurIPS 2025
  > Constrained Bayesian optimization (CBO) methods have seen significant success in black-box optimization with constraints.  One of the most commonly used CBO methods is the constrained expected impr...

**12. [CoT Information: Improved Sample Complexity under Chain-of-Thought Supervision](https://openreview.net/forum?id=OkVQJZWGfn)**
  *Awni Altabaa, Omar Montasser, John Lafferty*
  📍 NeurIPS 2025
  > Learning complex functions that involve multi-step reasoning poses a significant challenge for standard supervised learning from input-output examples. Chain-of-thought (CoT) supervision, which aug...

**13. [The Graphon Limit Hypothesis: Understanding Neural Network Pruning via Infinite Width Analysis](https://openreview.net/forum?id=EEZLBhyer1)**
  *Hoang Pham, The-Anh Ta, Tom Jacobs et al.*
  📍 NeurIPS 2025
  > Sparse neural networks promise efficiency, yet training them effectively remains a fundamental challenge. Despite advances in pruning methods that create sparse architectures, understanding why som...

**14. [Affine-Invariant Global Non-Asymptotic Convergence Analysis of BFGS under Self-Concordance](https://openreview.net/forum?id=d6UV0UNgn9)**
  *Qiujiang Jin, Aryan Mokhtari*
  📍 NeurIPS 2025
  > In this paper, we establish global non-asymptotic convergence guarantees for the BFGS quasi-Newton method without requiring strong convexity or the Lipschitz continuity of the gradient or Hessian. ...

**15. [On the necessity of adaptive regularisation: Optimal anytime online learning on $\boldsymbol{\ell_p}$-balls](https://openreview.net/forum?id=ONc9vWkwCp)**
  *Emmeran Johnson, David Martínez-Rubio, Ciara Pike-Burke et al.*
  📍 NeurIPS 2025
  > We study online convex optimization on $\ell_p$-balls in $\mathbb{R}^d$ for $p > 2$. While always sub-linear, the optimal regret exhibits a shift between the high-dimensional setting ($d > T$), whe...

**16. [Agnostic Learning under Targeted Poisoning: Optimal Rates and the Role of Randomness](https://openreview.net/forum?id=K3xaVpSHkV)**
  *Bogdan Chornomaz, Yonatan Koren, Shay Moran et al.*
  📍 NeurIPS 2025
  > We study the problem of learning in the presence of an adversary that can corrupt an $\eta$ fraction of the training examples with the goal of causing failure on a specific test point. In the reali...

**17. [Establishing Linear Surrogate Regret Bounds for Convex Smooth Losses via Convolutional Fenchel–Young Losses](https://openreview.net/forum?id=A4Xx9irvpp)**
  *Yuzhou Cao, Han Bao, Lei Feng et al.*
  📍 NeurIPS 2025
  > Surrogate regret bounds, also known as excess risk bounds,
bridge the gap between the convergence rates of surrogate and target losses. The regret transfer is lossless if the surrogate regret bound...

**18. [Minimax Adaptive Online Nonparametric Regression over Besov spaces](https://openreview.net/forum?id=nKuFQhKZtt)**
  *Paul Liautaud, Pierre Gaillard, Olivier Wintenberger*
  📍 NeurIPS 2025
  > We study online adversarial regression with convex losses against a rich class of continuous yet highly irregular competitor functions,% prediction rules, 
modeled by Besov spaces $B_{pq}^s$ with g...

**19. [An Efficient Orlicz-Sobolev Approach for Transporting Unbalanced Measures on a Graph](https://openreview.net/forum?id=VymXLPX6Ps)**
  *Tam Le, Truyen Nguyen, Hideitsu Hino et al.*
  📍 NeurIPS 2025
  > We investigate optimal transport (OT) for measures on graph metric spaces with different total masses. To mitigate the limitations of traditional $L^p$ geometry, Orlicz-Wasserstein (OW) and general...

**20. [Exploration via Feature Perturbation in Contextual Bandits](https://openreview.net/forum?id=gAddPMjmUc)**
  *Seouh-won Yi, Min-hwan Oh*
  📍 NeurIPS 2025
  > We propose *feature perturbation*, a simple yet effective exploration strategy for contextual bandits that injects randomness directly into feature inputs, instead of randomizing unknown parameters...

### 📌 Poster

**21. [The Rank and Gradient Lost in Non-stationarity: Sample Weight Decay for Mitigating Plasticity Loss in Reinforcement Learning](http://arxiv.org/abs/2604.01913v1)**
  *Zihao Wu, Hongyao Tang, Yi Ma et al.*
  > Deep reinforcement learning (RL) suffers from plasticity loss severely due to the nature of non-stationarity, which impairs the ability to adapt to new data and learn continually. Unfortunately, ou...

**22. [Tight analyses of first-order methods with error feedback](https://openreview.net/forum?id=hlPk6Hi43e)**
  *Daniel Berg Thomsen, Adrien Taylor, Aymeric Dieuleveut*
  📍 NeurIPS 2025 poster
  > Communication between agents often constitutes a major computational bottleneck in distributed learning. One of the most common mitigation strategies is to compress the information exchanged, there...

**23. [Towards Generalizable 3D Human Pose Estimation via Ensembles on Flat Loss Landscapes](https://openreview.net/forum?id=nZ4mFzCZIx)**
  *Jumin Han, Jun-Hee Kim, Seong-Whan Lee*
  📍 NeurIPS 2025 poster
  > 3D Human Pose Estimation (HPE) is a fundamental task in the computer vision. Generalization in 3D HPE task is crucial due to the need for robustness across diverse environments and datasets. Existi...

**24. [A geometric framework for momentum-based optimizers for low-rank training](https://openreview.net/forum?id=cCefuzQrjK)**
  *Steffen Schotthöfer, Timon Klein, Jonas Kusch*
  📍 NeurIPS 2025 poster
  > Low-rank pre-training and fine-tuning have recently emerged as promising techniques for reducing the computational and storage costs of large neural networks. Training low-rank parameterizations ty...

**25. [Iterative Missing Data Imputation with Model Form Adaptation and Non-Missing Feature Supervision](https://openreview.net/forum?id=L84DdFuvwV)**
  *Hao Wang, zhengnan li, Zhichao Chen et al.*
  📍 NeurIPS 2025 poster
  > Iterative imputation is a prevalent method for missing data imputation, where each feature is imputed iteratively by treating it as a target variable estimated from all other features. However, ite...

**26. [Exploring Landscapes for Better Minima along Valleys](https://openreview.net/forum?id=XxRKqFsvoK)**
  *Tong Zhao, Jiacheng Li, Yuanchang Zhou et al.*
  📍 NeurIPS 2025 poster
  > Finding lower and better-generalizing minima is crucial for deep learning. However, most existing optimizers stop searching the parameter space once they reach a local minimum. Given the complex ge...

**27. [How Compositional Generalization and Creativity Improve as Diffusion Models are Trained](https://openreview.net/forum?id=1OUEnfusEd)**
  *Alessandro Favero, Antonio Sclocchi, Francesco Cagnetta et al.*
  📍 ICML 2025 poster
  > Natural data is often organized as a hierarchical composition of features. How many samples do generative models need in order to learn the composition rules, so as to produce a combinatorially lar...

**28. [Rethinking Benign Overfitting in Two-Layer Neural Networks](https://openreview.net/forum?id=Uc0dTE2Wox)**
  *Ruichen Xu, Kexin Chen*
  📍 ICML 2025 poster
  > Recent theoretical studies (Kou et al., 2023; Cao et al., 2022) revealed a sharp phase transition from benign to harmful overfitting when the
noise-to-feature ratio exceeds a threshold—a situation ...

**29. [The Importance of Being Lazy: Scaling Limits of Continual Learning](https://openreview.net/forum?id=edhBkkYS8R)**
  *Jacopo Graldi, Alessandro Breccia, Giulia Lanzillotta et al.*
  📍 ICML 2025 poster
  > Despite recent efforts, neural networks still struggle to learn in non-stationary environments, and our understanding of catastrophic forgetting (CF) is far from complete.
In this work, we perform ...

**30. [Generalization of noisy SGD in unbounded non-convex settings](https://openreview.net/forum?id=Au9rfI6Fjd)**
  *Leello Tadesse Dadi, Volkan Cevher*
  📍 ICML 2025 poster
  > We study generalization of iterative noisy gradient schemes on smooth non-convex losses. Formally, we establish time-independent information theoretic generalization bounds for Stochastic Gradient ...

### 🆕 Recent (ArXiv / OpenReview)

**31. [Conservative flux reconstruction for an elliptic interface problem using CutFEM](http://arxiv.org/abs/2604.02153v1)**
  *Daniela Capatina, Aimene Gouasmi*
  > This paper deals with the local recovery of conservative fluxes for an elliptic interface problem with discontinuous coefficients. The transmission conditions on the interface are imposed weakly an...

**32. [Flux-equilibrated based a posteriori error analysis for an interface problem with CutFEM](http://arxiv.org/abs/2604.02137v1)**
  *Daniela capatina, Aimene Gouasmi*
  > This paper addresses the local recovery of conservative fluxes and the a posteriori error analysis for an elliptic interface problem with discontinuous coefficients. The transmission conditions on ...

**33. [A variationally consistent beam-to-beam point coupling formulation for geometrically exact beam theories](http://arxiv.org/abs/2604.02049v1)**
  *Ivo Steinbrecher, Nora Hagmeyer, Christoph Meier et al.*
  > Slender beam-like structures frequently occur in engineering applications and often interact at discrete locations through joints or connectors. Accurate modeling of such interactions is particular...

**34. [Tree-Adaptive Multiscale Kernel Lasso in Samplet Coordinates](http://arxiv.org/abs/2604.01794v1)**
  *Sara Avesani, Gaia Fumagalli, Michael Multerer et al.*
  > We develop a novel framework for sparse multiscale kernel approximation of large scattered data problems based on a samplet representation. Samplets form a multiresolution analysis of localized dis...

**35. [Attention Mechanisms Through the Lens of Numerical Methods: Approximation Methods and Alternative Formulations](http://arxiv.org/abs/2604.01757v1)**
  *Michel Fabrice Serret, Alice Cortinovis, Yijun Dong et al.*
  > The attention mechanism is the computational core of modern Transformer architectures, but its quadratic complexity in the input sequence length is the bottleneck for large-scale inference. This ha...

**36. [The Landau--Lifshitz--Bloch equation with spin diffusion: Global strong solution and finite element approximation](http://arxiv.org/abs/2604.01668v1)**
  *Agus L. Soenjaya*
  > The spin-diffusion Landau--Lifshitz--Bloch (SDLLB) system is a nonlinearly coupled system of quasilinear vector-valued PDEs which models the interaction between spin-polarised currents and magnetis...

**37. [Random Coordinate Descent on the Wasserstein Space of Probability Measures](http://arxiv.org/abs/2604.01606v1)**
  *Yewei Xu, Qin Li*
  > Optimization over the space of probability measures endowed with the Wasserstein-2 geometry is central to modern machine learning and mean-field modeling. However, traditional methods relying on fu...

**38. [A Determinantal Approach to a Sharp $\ell^1-\ell^\infty-\ell^2$ Norm Inequality](http://arxiv.org/abs/2604.01525v1)**
  *Jose Antonio Lara Benitez*
  > We give a short linear--algebraic proof of the inequality \[ \|x\|_1\,\|x\|_\infty \le \frac{1+\sqrt{p}}{2}\,\|x\|_2^2, \] valid for every \(x\in\mathbb{R}^p\). This inequality relates three fundam...
